soccer/Robot.hpp

```diff
@@ -530,6 +532,9 @@ class OurRobot : public Robot {
     friend class MotionControl;
 
 private:
+    mutable RJ::Time _lastBallSense;
```

Comment:
Oh, I see now. Um, it would be best to move the place where you update it to setRadioRx or somewhere in processor. hasBall really shouldn't be relied on to be called in order to set it and it should really be based on when we receive the information instead of when we call hasBall which I don't think is guarrenteed 100% to be called every frame